packages/check/src/main.rs

```diff
@@ -1,3 +1,8 @@
+//! CosmWasm is a smart contract platform for the Cosmos ecosystem.
+//! This binary provides a CLI tool for verifying CosmWasm smart contracts before uploading to a blockchain.
+//!
+//! For more information, see: <https://docs.cosmwasm.com>
```

Comment:
The docs were recently moved to `https://cosmwasm.cosmos.network`, so I would link to that.